modules/fee-market/src/s2s/payment.rs

```diff
@@ -112,7 +112,7 @@ where
 		// Pay treasury_sum reward
 		do_reward::<T, I>(
 			relayer_fund_account,
-			&T::TreasuryPalletId::get().into_account(),
+			&T::TreasuryPalletId::get().into_account_truncating(),
```

Comment:
what't the difference?

Reply:
See https://github.com/paritytech/substrate/pull/10719